### ISSUE TITLE
Wallabag and a self-signed Wallabag certificate

### ISSUE THREAD

**Issue by nukedone** [jangernert/FeedReader#914]
Firstly, thanks for such a great peace of software. Probably the best RSS reader out there.

I managed to connect my feeds to belonging services. I have a problem with sharing.
I own a self-hosted Wallabag server which is served with the self-signed certificate. The app refuses to connect to the service. 
It would be great if I could still use Wallabag sharing and not leave my private computer space.

**In other words - issue: allow self-signed instances of Wallabag to connect to the FeedReader.**

**Comment by brendanlong** [comment]
To use a self-signed certificate, you would need to make GNOME trust it. I think the instructions [on this StackOverflow question](https://unix.stackexchange.com/questions/90450/adding-a-self-signed-certificate-to-the-trusted-list) should do it.

Another option would be to connect to Wallabag over HTTP instead of HTTPS if you're not concerned about the security (on a highly trusted local network if you don't mind other users knowing your password).

If your Wallabag server has a host name, I'd recommend getting a CA-signed certificate from Let's Encrypt (free).

On the FeedReader side, it might be good to add a prompt to let people add a certificate to the local trust store. We probably don't want to add an option to turn off cert validation. We would also need to handle the case where a cert is replaced, and we don't currently have a way to prompt to trust the new one. The logic here may be more work than we're likely to take on any time soon, so one of the workarounds above is probably going to be the only option for now.
